src/puppet-padplus.ts

```diff
@@ -1206,14 +1206,14 @@ export class PuppetPadplus extends Puppet {
   roomAnnounce (roomId: string): Promise<string>
   roomAnnounce (roomId: string, text: string): Promise<void>
   async roomAnnounce (roomId: any, text?: any): Promise<string | void> {
-    log.silly(PRE, `roomId : ${util.inspect(roomId)}, text: ${text}`)
+    log.silly(PRE, `roomId : ${util.inspect(roomId)}, text: ${text || ''}`)
```

Comment:
Let's remove `uitl.inspect()` in this log. This is a problem left over by history.

Reply:
I think `${text}` is the same to `${text || ''}` here, no use for empty announce log.